test/spechelper.js

```diff
@@ -299,4 +300,15 @@ exports.shouldTestAddOn = function (addOn) {
   return cldTestAddons.trim().split(',').includes(addOn.toLowerCase())
 }
 
+/**
+ * Should a certain feature be tested?
+ *
+ * @param {string} feature The feature to test.
+ *
+ * @return boolean
+ */
+exports.shouldTestFeature = function(feature){
+  return feature === 'ALL';
```

Comment:
I would expect something like with testing addons above ^... this function always returns false in the code...

Reply:
updated